### DISCUSSION TITLE
Motion only videos

### DISCUSSION THREAD

**Discussion by jmtatsch** [blakeblackshear/frigate#14246] [Ideas]
Could we get an option for motion only videos?
Really like the visualization there https://youtu.be/NSS6yAMZF78?si=NWhVj5ZQFkc8VDO1
I think that might make reviewing detections/motion tabs faster and would also be great for debugging.

**Comment by NickM-27** [comment]
we've look at doing this for motion review. It likely wouldn't be a video but rather an image showing all the motion in each hour. It is still an early idea

**Reply by jmtatsch** [reply]
Could also be the frame with the most motion.
